javascript/index.js

```diff
@@ -0,0 +1,316 @@
+
+var stack = [];
+
+function countNrDecimals(num) {
+  let flag = false;
+  let nrDec = 0;
+
+  try{
+    for(var i of num) {
+      if(flag) {
+        nrDec++;
+      }
+      if(i == ".") {
+        flag = true;
+      }
+    }
+  }
```

Comment:
Function countNrDecimals is used in the code only to check if a number (num) contains the decimal point.
So only the flag is important, nrDec can be deleted and the function can be simplified and made more efficient!